midterm-2023-11-06/code/5215-berat-useini/script.ts

```diff
@@ -0,0 +1,191 @@
+type RockBand = {
```

Comment:
The interface are more mapped to the data we will display, than to the data we actually receive +5